src/containers/AskMeBot/AskMeBot.tsx

```
@@ -0,0 +1,213 @@
+              .filter((i) => !i.prompt)
+              .map((message) => (
+                <div className={`${message.role === 'system' ? styles.System : styles.User}`}>
+                  <Markdown>{message.content}</Markdown>
+                </div>
+              ))}
+            {isLoading && <div className={styles.Loader} />}
+          </div>
+          <div className={styles.SuggestionsContainer}>
+            {messages.length <= 1 &&
+              quickSuggestions.map((suggestion, index) => (
+                <div onClick={() => setMessage(suggestion)} className={styles.Suggestion}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)